### ISSUE TITLE
a type definition containing  a "'"  will cause syntax error

### ISSUE THREAD

**Issue by newbienewbie** [sequelize/sequelize-auto#100]
e.,g.  when there's a column type: 

``` SQL
create table `a_table`(
    # ...
    `g` set('A','B') DEFAULT NULL , 
    # ...
)ENGINE=InnoDB DEFAULT CHARSET=utf8;

```

 within a MySQL databse  , the corresponding code  generated by the `sequelize-auto` tool  is : 

``` JavaScript
    g: {
      type: 'SET('A','B')',
      allowNull: true
    },
```

**Comment by francisgendreau** [comment]
bug originate from Sequelize not having a SET() datatype method. I worked around that by forcing SET to be replaced by ENUM in lib/index.js at around line 210.

**Comment by matchdav** [comment]
I ran into this too.

I got around it by wrapping it in " rather than '.  Maybe your linter won't like it, but it does seem to get around the problem where maybe other types aren't supported either.

**Comment by steveschmitt** [comment]
Fixed in 0.6.1
